funnel/models/event.py

```diff
@@ -221,12 +234,7 @@ def sync_from_list(cls, ticket_list, space, ticket_client=None):
         for ticket_dict in ticket_list:
             ticket = SyncTicket.fetch(ticket_dict.get('ticket_no'), ticket_dict.get('order_no'), space)
             ticket_ticket_type = TicketType.get(ticket_dict.get('ticket_type'), space, create=True)
-            # get or create participant
-            ticket_participant = Participant.query.filter_by(email=ticket_dict.get('email'), proposal_space=space).first()
-            if not ticket_participant:
-                # create a new participant record if required
-                ticket_participant = Participant.make_from_dict(ticket_dict, space)
-                db.session.add(ticket_participant)
+            ticket_participant = Participant.get(space, ticket_dict.get('email'), fields=ticket_dict, create=True)
```

Comment:
`ticket_dict.get('email')` isn't normalising for case. Email is case-insensitive but the database is case-sensitive. You have to force lowercase it.